docker-compose.yml

```diff
@@ -191,6 +191,17 @@ services:
       - FLINKSTER_CLIENT_ID=${FLINKSTER_CLIENT_ID:?missing/empty}
       - FLINKSTER_SECRET=${FLINKSTER_SECRET:?missing/empty}
     restart: unless-stopped
+    healthcheck:
+      test: ps aux | grep -q 'x2gbfs\.x2gbfs' || exit 1
+      # If you want to wait until all feeds have been generated 
+      # before depending lamassu starts subscribing them, 
+      # configure an appropriate startup_period, e.g. 60s
```

Comment:
Is it correct that the actual time needed to initially generate all feeds depends on how long the *slowest of an arbitrary number* of feeds needs to respond?

I think it should be stated here that the value should be picked based on the upstream providers queried by x2gbfs.